tests/FSharp.Compiler.ComponentTests/EmittedIL/Inlining/Match01_RealInternalSignatureOff.fs.il.netcore.release.bsl

```diff
@@ -1107,6 +1099,42 @@
     } 
   } 
 
+  .method public static int32  select1(class Match01/Test1 x) cil managed
```

Comment:
That doesn't look good, that it changed ordering. A potential determinism issue here